README.md

```diff
@@ -1,22 +1,22 @@
-## Google Analytics for Firebase Kit Integration
+## Firebase for Google Analytics 4 (GA4) Kit Integration
 
-This repository contains the [Google Analytics for Firebase](https://firebase.google.com/docs/analytics/android/start) integration for the [mParticle Android SDK](https://github.com/mParticle/mparticle-android-sdk).
+This repository contains the [Firebase for GA4](https://firebase.google.com/docs/analytics/get-started?platform=android) integration for the [mParticle Android SDK](https://github.com/mParticle/mparticle-android-sdk).
 
 ### Adding the integration
 
 1. Add the kit dependency to your app's build.gradle:
 
     ```groovy
     dependencies {
-        implementation 'com.mparticle:android-googleanalyticsfirebase-kit:5+'
+        implementation 'com.mparticle:android-googleanalyticsfirebasega4-kit:5+'
     }
     ```
 2. Follow the mParticle Android SDK [quick-start](https://github.com/mParticle/mparticle-android-sdk), then rebuild and launch your app, and verify that you see `"Google Analytics for Firebase detected"` in the output of `adb logcat`.
 3. Reference mParticle's integration docs below to enable the integration.
 
 ### Documentation
 
-[Google Analytics for Firebase integration](http://docs.mparticle.com/integrations/firebase)
+[Firebase for GA4 integration](http://docs.mparticle.com/integrations/google-analytics-4/event/)
```

Comment:
is http://docs.mparticle.com/integrations/google-analytics-4/event/ online yet?  i see an error page

Reply:
ugh, I hate to throw this in last minute, but would it be on the table to just update the kit name/dependency name to `firebasega4-kit` instead of `googleanalyticsfirebasega4-kit` the current/latter one is just so painful to type lol...it has to be annoying to the client on some level

Reply:
i'm indifferent to the name as long as it compiles successfully...i think Rob just expanded on older convention.

Reply:
yea, my gut says it's definitely just throwing a wrench in the works right now...I'm all good with the name we have now